src/ray/rpc/test/grpc_bench/grpc_bench.cc

```diff
@@ -15,8 +15,8 @@
 #include "ray/common/asio/instrumented_io_context.h"
 #include "ray/rpc/grpc_server.h"
 #include "ray/rpc/server_call.h"
-#include "src/ray/rpc/test/grpc_bench/helloworld.grpc.pb.h"
-#include "src/ray/rpc/test/grpc_bench/helloworld.pb.h"
+#include "ray/rpc/test/grpc_bench/helloworld.grpc.pb.h"
+#include "ray/rpc/test/grpc_bench/helloworld.pb.h"
```

Comment:
Do we need to change these?

Reply:
reverted. but looks like `bazel build //src/ray/rpc/test/grpc_bench:grpc_bench` fail to build on master already.